tests/infra/remote.py

```diff
@@ -451,8 +451,9 @@ def _setup_files(self):
             src_path = os.path.normpath(os.path.join(os.getcwd(), path))
             assert self._rc("ln -s {} {}".format(src_path, dst_path)) == 0
         for path in self.data_files:
-            dst_path = os.path.join(self.root, os.path.basename(path))
-            self.cp(path, dst_path)
+            if len(path) > 0:
```

Comment:
Why is this check here? We shouldn't be passing empty paths, and this seems to be hiding that?

Reply:
This was for the `--config` option, as I set the default to empty string. I will try removing this option, as it is not required in my case

Reply:
@fotiskoun this is fine for now, but please create a ticket with the details of what needs to be addressed.